sn_node_manager/src/bin/cli/main.rs

```diff
@@ -148,6 +149,9 @@ pub enum SubCmd {
     Daemon(DaemonSubCmd),
     #[clap(subcommand)]
     Faucet(FaucetSubCmd),
+    /// Manage RPC client commands
+    #[clap(name = "rpc", subcommand)]
+    Restart(RpcSubCmd),
```

Comment:
This should be named `Rpc` rather than `Restart`.

Reply:
good shout